Documentation/contributor-guide/roadmap.md

```diff
@@ -0,0 +1,47 @@
+# roadmap
+
+etcd uses GitHub milestones to track all tasks in each major or minor release. The roadmap.md file only records the most
+important tasks for each release.
+Please also refer to [here](https://docs.google.com/spreadsheets/d/1hFpIbkI0IDum0SPy8txj7bVHxQRrAB_82f1Yqz_Pa8M/edit#gid=0&fvid=1685003197)
+for more context. Note that etcd will continue to focus on technical debt over the next few major or minor releases.
+
+## v3.6.0
+
+For a full list of tasks in v3.6.0, please see [milestone etcd-v3.6](https://github.com/etcd-io/etcd/milestone/38).
+
+| Title                                                              | Priority | Note                                                                                                         |
+|--------------------------------------------------------------------|----------|--------------------------------------------------------------------------------------------------------------|
+| Support downgrade                                                  | P0       | etcd will support downgrade starting from 3.6.0. But it will also support offline downgrade from 3.5 to 3.4. |
+| StoreV2 deprecation                                                | P0       | This task will be covered in both 3.6 and 3.7.                                                               |
+| Evaluate and (Gradulate or deprecate/remove) experimental features | P2       | This task will be covered in both 3.6 and 3.7.                                                               |
+| Support /livez and /readyz endpoints                               | P1       | It provides clearer APIs, and can also workaround the stalled writes issue                                   |
+| Bump gRPC                                                          | P0       | It isn't guaranteed to be resolved in 3.6, and might be postponed to 3.7 depending on the effort and risk.   |
```

Comment:
I would not mark it as P0 if we are considering it to be postponed to v3.7. Sounds more like P1. Maybe checking if it can be postponed is a P0, but that's to granular for this list.

Reply:
updated & thx